### ISSUE TITLE
Mocha exploded when generating coverage reports

### ISSUE THREAD

**Issue by joanvila** [pghalliday/grunt-mocha-test#129]
I'm following the readme to generate the coverage reports. I have a grunt task like this:

```
mochaTest: {
    test: {...},
    coverage: {...}
}
```

As it is in the readme.

I also created the blanket.js file changing the src dir to the lib dir as I am testing the js files that I have inside the lib dir.

The problem comes when I run the grunt task. It says to me:
```
Running "mochaTest:test" (mochaTest) task
>> Mocha exploded!
>> SyntaxError: Unexpected token (3:6)
>> ...
```

The problem seems to be in a javascript test file. In a require. There, I'm requiring the actual js file from the lib folder to test.

Any thoughts on this? Thanks :)

**Comment by pghalliday** [comment]
I don't know if you copy pasted or if this is a transcribing issue but

```javascript
require(../lib/jsfile.js)
```

should be

```javascript
require('../lib/jsfile.js')
```

note the quotes as the path should be a string

**Comment by joanvila** [comment]
Yes, sorry, I was writting it manually here and I forgot the quotes. It's actually `const endpointBuilder = require('../lib/endpoint-builder');`. Also without the js extension.

**Comment by joanvila** [comment]
Looking at the open PR in the repo, I've seen some discussion around the `'use strict'`. I'm using it everywhere. Is this a problem? I'm also using the `const` and `let` keywords for defining variables.

Thanks for your help

**Comment by pghalliday** [comment]
I don't think it would be related to `use strict`. And yes `html-cov` is no longer supported. I have been migrating everything to `istanbul` for coverage. At some point I'll change the docs here to recommend `istanbul` instead

**Comment by joanvila** [comment]
Perfect, meanwhile I'm using Istanbul with it's commands as a workaround while `grunt-mocha-test` is being updated :)

**Comment by yuchaosydney** [comment]
Can someone show a demo on how to generate coverage report with`istanbul`?
